### ISSUE TITLE
Error executing Migrations

### ISSUE THREAD

**Issue by zhiwenchong** [volkanceylan/Serenity#4926]
I've tried to use Serenity many times in the past but have never once gotten it work out of the box -- and still haven't. I'm using Visual Studio 2019. I'm just wondering what setup I need to have to have it work out of box.

1. I download Serene and start a new project with the Serene ASP.NET Core template. I press F5 to compile, it complains ASP.NET Core 2.2 SDKnot found. I go to the website to download, but it says its been deprecated.
2. So I follow this guide (https://github.com/volkanceylan/Serenity/issues/4801) and to upgrade my Serene ASP.NET Core 2.2 project to ASP.NET Core 3.1. Press F5, it compiles, but I hit this unresolved issue:
https://github.com/volkanceylan/Serenity/issues/4801#issuecomment-582151311
3. So I gave up and hunted around for ASP.NET Core 2.2 SDK. Downloaded and installed. Started a new Serene ASP.NET Core project. Pressed F5. Error on "Error executing migrations" in DataMigrations.cs.

Just curious, what version of Visual Studio/ASP.NET Core do I need to get started without running into issues? I just want to explore the functionality without hitting a compile brickwall every time. Thanks.

**Comment by minhhungit** [comment]
1. just install .net core 2.2 (after it works you can upgrade to 3.1 if you want) or you can use .net framework
2. About "Error executing migrations, try to create an empty database with db name same with your connection string and run again, it should work

If not please show more detail about your exception

My projects are using either .net framework or .net core 2.2 or .net core 3.1 with visual studio 2019

**Comment by minhhungit** [comment]
btw what is kind of database you are using, MSSQL, Oracle or MySql... ?
take a look this https://serenity.is/docs/setting-connection-dialect

**Comment by minhhungit** [comment]
And there are a lot video tutorial here:

https://github.com/volkanceylan/Serenity#video-tutorials-thanks-a-lot-to-samdubey-and-sayuga-for-these-contributions

**Comment by zhiwenchong** [comment]
Thanks!

I didn't specify any database because Serene was supposed to create tables in LocalDB. (I'm running VS2019/ASP.NET Core 2.2.8) I just tried it again -- my steps were literally Start New Project > SERENE (ASP.NET Core) > (install AdminLTE, etc.) OK > F5. 

But it works now! (not sure why it didn't before? It was failing on migrations. I wiped out everything and started again)

I guess the golden configuration is .NET Core 2.2 SDK before installing SERENE, and things should work out of box.
